CUP-parser/AST/Abstract/BinaryOp.java

```diff
@@ -0,0 +1,16 @@
+package AST.Abstract;
+
+import AST.Master.Node;
+
+public abstract class BinaryOp implements Node{
+    public BinaryOp(Node left, Node right, String operator) {
+        Left = left;
+        Right = right;
+        Operator = operator;
+    }
+
+    public String NodeType;
```

Comment:
How come NodeType is not set in the constructor?

Reply:
@christopher-jensen